### PR TITLE
Update the French translation

### PR DESCRIPTION
**Pull request by zenseii**

### CODE REVIEW

File: files/lang/fr.po [hunk 1]
```diff
@@ -2456,9 +2457,8 @@ msgstr "Recruter des créatures"
 msgid "Max"
 msgstr "Max"
 
-#, fuzzy
 msgid "Hire all creatures in the town."
-msgstr "Perdre tous les héros et tous les villages."
+msgstr "Recruter tous les créatures disponibles dans la village."
```
Comment (zenseii) [inline files/lang/fr.po:2461]:
```suggestion
msgstr "Recruter tous les créatures disponibles dans le village."
```

File: files/lang/fr.po [hunk 2]
```diff
@@ -2755,18 +2759,15 @@ msgstr "Paramètres de jeu expérimentaux."
 msgid "Toggle instant battle mode."
 msgstr "Change les modes de combat instantané."
 
-#, fuzzy
 msgid "Check and configure all the hot keys present in the game."
-msgstr "Verifier les raccourcirs clavier du jeu."
+msgstr "Verifier et configurer les raccourcirs clavier du jeu."
```
Comment (zenseii) [inline files/lang/fr.po:2763]:
```suggestion
msgstr "Verifier et configurer les raccourcis clavier du jeu."
```

**Comment by zenseii** [comment]
Hi @ihhub.

This PR is ready for your review. We've had 3 French native speakers look at it so it should be good.

**Comment by ihhub** [comment]
@zenseii , a bunch of thanks for updating French translation!

**Pull request merged**